CHANGELOG.md

```diff
@@ -48,13 +42,18 @@ Security improvements of container image and enhancements for the `ReferringConn
 - Run java process with a non-root user
 
 ### Deployment Migration Notes
+- `default` datasource has to be added
+  - `EDC_DATASOURCE_DEFAULT_NAME`=default
+  - `EDC_DATASOURCE_DEFAULT_URL`=jdbc:postgresql://connector:5432/edc
+  - `EDC_DATASOURCE_DEFAULT_USER`=user
+  - `EDC_DATASOURCE_DEFAULT_PASSWORD`=password
```

Comment:
Convention: We should have just added those to our .env file, right?

Reply:
Indeed they have to be included to the .env file. These changes already where included in the main branch